packages/tab/src/css/index.js

```diff
@@ -44,31 +44,21 @@ export default {
     position: 'relative',
     width: '100%',
     height: '64px',
-    overflow: 'hidden',
+    overflow: 'hidden'
+  },
+  [`.psds-tab__list.psds-theme--${themeNames.light}`]: {
+    borderBottom: `1px solid ${core.colors.gray02}`,
     '&:focus': {
       outline: 'none',
-      borderBottomWidth: '4px',
-      // __text
-      '& > div': {
-        height: 'calc(100% + 4px)',
-        marginBottom: '-4px'
-      },
-      // __bar
-      '& span': {
-        bottom: '-1px'
-      },
-
-      // __overflow-button
-      '& > button': {
-        height: 'calc(100% + 4px)'
-      }
+      boxShadow: '0 0 0 2px rgba(0, 123, 255, .5)'
     }
   },
-  [`.psds-tab__list.psds-theme--${themeNames.light}`]: {
-    borderBottom: `1px solid ${core.colors.gray02}`
-  },
   [`.psds-tab__list.psds-theme--${themeDefaultName}`]: {
-    borderBottom: `1px solid ${core.colors.gray04}`
+    borderBottom: `1px solid ${core.colors.gray04}`,
+    '&:focus': {
```

Comment:
now that the focus state is the same for all themes you could define these styles once on the `.psds-tab__list` selector